resources/scripts/jquery/jquery-utils.js

```diff
@@ -308,6 +308,30 @@
     	$(".eXist_wizard_next", container).click(function () {
     		if (currentPage == pages.length - 1)
     			return;
+		var name = document.getElementsByName('name')[0].value;
```

Comment:
You can use `const` instead of `var` here for all three values.

Reply:
Agreed, I also prefer const, I'm not sure why I used var :) !